js/main.js

```diff
@@ -172,6 +172,7 @@ button.forEach((btn) => {
   btn.addEventListener('click', Modalopen);
```

Comment:
[ OPTIONAL ] Please add unnecessary files and folders like `vscode` to `.gitignore`